src/lib/coverage-pool/coverage-pools.js

```diff
@@ -0,0 +1,141 @@
+import { Token } from "../liquidity-rewards/utils"
```

Comment:
Let's move `Token` class maybe to `lib/helper` ?

Reply:
Fixed in 987f8f175e8697d904f03c5c2c805f32ce34d7bc